### DISCUSSION TITLE
Question: Will Week 7 Source Code ever be released?

### DISCUSSION THREAD

**Discussion by chonkyman** [FunkinCrew/Funkin#1545] [General]
Hey ninjamuffin, or anyone else who's reading this. I hope you're all having a nice morning/evening.

Will week 7 source code ever be released?

Lots of people have been waiting for a while now.

Week 7 changes loads of things, it's not just a few minor tweaks to the code that can be done in an half an hour or so.

Last update to the code was months ago.

**Comment by steviegt6** [comment]
Don't quote me on this as I'm not sure, but I'd imagine the source code will be committed whenever week 7 is added to Itch.io. Currently, week 7 is labeled as a Newgrounds-exclusive "preview" (last I checked, anyway).

**Comment by oketariq** [comment]
Ninjamuffin is currently working on getting the cutscenes working on pc, so once that is done they should release the game on itch.io and release the source on github! :)

**Comment by foxsouns** [comment]
> Ninjamuffin is currently working on getting the cutscenes working on pc, so once that is done they should release the game on itch.io and release the source on github! :)

hell yeah! cant wait hehe

**Comment by ghost** [comment]
the source is on github somewhere but i forgot where

**Comment by ghost** [comment]
you can also enter devtools and click sources

**Comment by wellsilver** [comment]
Week 7 source code will most likely be released as soon as it is no longer a pre-view on newgrounds.

**Comment by ghost** [comment]
> the source is on github somewhere but i forgot where

Actually, the **compiled version** is on GitHub. The source code itself has not been released.

**Comment by Tysonhutchinson** [comment]
> Ninjamuffin is currently working on getting the cutscenes working on pc, so once that is done they should release the game on itch.io and release the source on github! :)

HELL YEAH but thats the question when is it gonna be on itch.io

**Comment by ghost** [comment]
when its on github

**Comment by foxsouns** [comment]
> HELL YEAH but thats the question when is it gonna be on itch.io

@Tysonhutchinson Github source released means we can compile it for ourselves, and can run it on whatever we want. This means Windows, Linux, and MacOS binaries, along with html stuff, all in our hands to make. We are assuming that with this, NinjaMuffin is going to update the web stuff for itch.io as well, but it doesn't really matter; we can run it on any computer we want at that point.
